.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  tests:
+    name: Composable Test
+    #container:
+    #  image: composablefi/composable-sandbox:latest
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build the stack
+      run: docker-compose up -d && cargo test
```

Comment:
We need to wait till the container completely starts all the nodes.

The last line is usually **polkadot launch completed**

Reply:
I did this https://github.com/ComposableFi/beefy-client/pull/9